src/views/projectExplorer/index.ts

```diff
@@ -82,6 +84,15 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
           await commands.executeCommand(`code-for-ibmi.setDeployLocation`, undefined, element.workspaceFolder, `${element.description}`);
         }
       }),
+      commands.registerCommand(`vscode-ibmi-projectexplorer.projectExplorer.iprojShortcut`, async (element: Source) => {
+        if (element) {
+          const iProject = ProjectManager.get(element.workspaceFolder);
+
+          const fPath = iProject!.getIProjFilePath();
```

Comment:
Instead of using `iProject!`, we usually check if `iProject` is defined just in case.